lighthouse-cli/run.ts

```diff
@@ -29,28 +29,23 @@ interface LighthouseError extends Error {
   code?: string
 }
 
-// Boolean values that are true are rendered without a value (--debug vs. --debug=true)
-function formatArg(arg: string, value: any): string {
-  if (value === true) {
-    return `--${arg}`
-  }
-
-  // Only quote the value if it contains spaces
-  if (value.indexOf(' ') !== -1) {
-    value = `"${value}"`
-  }
-
-  return `--${arg}=${value}`
-}
-
 // exported for testing
 export function parseChromeFlags(flags: string) {
-  let args = yargs(flags).argv;
-  const allKeys = Object.keys(args);
-
-  // Remove unneeded arguments (yargs includes a _ arg, $-prefixed args, and camelCase dupes)
-  return allKeys.filter(k => k !== '_' && !k.startsWith('$') && !/[A-Z]/.test(k))
-      .map(k => formatArg(k, args[k]));
+  const parsed = yargsParser(
+      flags || '', {configuration: {'camel-case-expansion': false, 'boolean-negation': false}});
```

Comment:
for `flags` here, seems like this is broken type checking? If it's a string the only falsy value is already `''`

could either initialize a default value in `getDebuggableChrome` or do `flags = ''` as the parameter default